handlers/handlers_test.go

```diff
@@ -67,15 +67,15 @@ func TestUnitHandlers(t *testing.T) {
 		mockConfig := config.Config{}
 
 		router := mux.NewRouter()
-		router.HandleFunc(url, Bulletin(mockConfig, mockRenderClient, mockZebedeeClient, mockArticlesApiClient))
+		router.HandleFunc(url, SixteensBulletin(mockConfig, mockRenderClient, mockZebedeeClient, mockArticlesApiClient))
 
 		w := httptest.NewRecorder()
 
 		Convey("it returns 200 when rendered succesfully", func() {
-			mockArticlesApiClient.EXPECT().GetLegacyBulletin(ctx, accessToken, collectionID, lang, url).Return(&b, nil)
+			mockArticlesApiClient.EXPECT().GetLegacyBulletin(ctx, accessToken, collectionID, lang, "").Return(&b, nil)
 			mockZebedeeClient.EXPECT().GetBreadcrumb(ctx, accessToken, collectionID, lang, b.URI)
 			mockRenderClient.EXPECT().NewBasePageModel()
-			mockRenderClient.EXPECT().BuildPage(w, gomock.Any(), "bulletin")
+			mockRenderClient.EXPECT().BuildPage(w, gomock.Any(), "sixteens-bulletin")
```

Comment:
We probably should change the description of this convey from `test GetBulletin` to `test SixteensBulletin`